### ISSUE TITLE
关于loss的一些小疑问

### ISSUE THREAD

**Issue by LinPeiMing** [FVL2020/SC-NAFSSR#2]
您好：
loss += 0.0025 * (loss_photo + 0.1 * loss_smooth + loss_cycle)
这个0.0025是什么意思呢？好像并没有在论文中读到诶，是用来监督SCAM的公式4么？那为什么是0.0025呢？
还有个小问题就是看代码里用的是 num_blks = 32，width = 32，而把 num_blks = 128 width = 128注释掉了？但论文说用的是NAFSSR-L，有点没看懂最后用的哪个。
